notify/util.go

```diff
@@ -90,7 +90,7 @@ func Truncate(s string, n int) (string, bool) {
 	if n <= 3 {
 		return string(r[:n]), true
 	}
-	return string(r[:n-3]) + "...", true
+	return string(r[:n-1]) + "…", true
```

Comment:
"…" is encoded as 3 bytes in utf-8, so this doesn't save anything, and makes string longer by 2 bytes compared to previous version.

Furthermore blindly cutting last 3 or 1 byte can actually break the utf-8 sequence of bytes. Ideally string would be truncated at rune boundary.

Reply:
I fixed this eventually: https://github.com/prometheus/alertmanager/blob/main/notify/util.go#L88-L130

Reply:
Oh nice, I missed that when checking changelog of alertmanager 0.26.0.

Reply:
Thanks!